topics/imaging/tutorials/process-image-bioimageio/tutorial.md

```diff
@@ -0,0 +1,194 @@
+| 🧫 HPACellSegmentationModel | Cell segmentation | Immunofluorescence | Human Protein Atlas | [View model](https://bioimage.io/#/?tags=hpa&id=10.5281%2Fzenodo.6200635) |
+| 🧪 MitochondriaEMSegmentationBoundaryModel | Mitochondria segmentation | Electron microscopy | Human | [View model](https://bioimage.io/#/?id=10.5281%2Fzenodo.5874841) |
+
+## Model-specific example
+
++Here we illustrate the type of information that is both useful for understanding the model's biological context and necessary for using the Galaxy tool — specifically, the input axes and input size parameters.
+
++As an example, we consider the following model: **🧬 NucleiSegmentationBoundaryModel**
+
+This model segments nuclei in fluorescence microscopy images. It predicts <em>boundary maps</em> and <em>foreground probabilities</em> for nucleus segmentation, primarily in images stained with DAPI. The outputs are designed to be post-processed with methods such as Multicut or Watershed to achieve instance-level segmentation (object-based segmentation).
+
+- **Imaging modality**: Fluorescence microscopy  
+- **Task**: Nucleus segmentation (boundary-aware)  
+- **Input axes**: `bcyx`  
+- **Input size**: `256,256,1,1`  
+- **Model link**: [View on BioImage.IO](https://bioimage.io/#/r/ilastik/stardist_dsb_training_data)  
+- **Citation**: [10.5281/zenodo.5764893](https://doi.org/10.5281/zenodo.5764893)
+
+
+> <tip-title> Where to find this information on BioImage.IO </tip-title>
+>
+> You can find similar details for other models directly on [BioImage.IO](https://bioimage.io) by viewing each model’s card. Look under the “inputs” section of the RDF file to find the required `axes` and `input size` values. These parameters are essential for running the model correctly in Galaxy.
+>
+{: .tip}
+
+
+## Get data
+
+> <hands-on-title> Data Upload </hands-on-title>
+>
+> 1. Create a new history for this tutorial
+>
+> 2. Download the following image and import it into your Galaxy history.  
+>    For the purpose of this tutorial, we will use one image to test only one of the 11 available models:
+>
+>    - [`test_image_nuclei.tif`](../../images/process-image-bioimageio/input_nucleisegboundarymodel.png)
+>
+>    If you are importing the image via URL:
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+>    If you are importing the image from the shared data library:
+>
+>    {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
+>
+> 3. Rename the datasets appropriately if needed (e.g. `"BioImage.IO model"`, `"Test image"`)
+>
+> 4. Confirm the datatypes are correct (`pt` for the model, `tiff` or `png` for the image)
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="datatypes" %}
+>
+> 5. Import the BioImage.IO model from the Galaxy file repository:
+>
+>    - Click on **Upload Data**
+>    - Go to the **Choose remote files** tab
```

Comment:
I think it's called `Choose from repository`?
```suggestion
>    - Go to the ** Choose from repository** tab
```
<img width="882" alt="Bildschirmfoto 2025-04-04 um 13 06 08" src="https://github.com/user-attachments/assets/dc9aeacc-0d6a-4d62-acf5-be3f8a8850cb" />

Reply:
Yes, we are pushing for this less technical wording now.